resources/views/layouts/title.blade.php

```diff
@@ -0,0 +1,9 @@
+<title>
+    @hasSection('title')
+        {{ Rapidez::config('design/head/title_prefix') }}
+        @yield('title')
+        {{ Rapidez::config('design/head/title_suffix') }}
+    @else
+        {{ Rapidez::config('design/head/default_title') }}
```

Comment:
just in case, this should not be prefixed and suffixed right?

Reply:
Correct, the prefix and suffix only come into play when you have a title set. This follows the standard Magento behavior.